spek-ide-plugin-intellij-idea/build.gradle.kts

```diff
@@ -40,10 +40,17 @@ val buildMatrix = mapOf(
         "IJ183",
         ij.VersionRange("211.1", "211.*"),
         arrayOf("java", "org.jetbrains.kotlin:211-1.4.21-release-IJ6556.4")
+    ),
+	"IJ212" to ij.BuildConfig(
+        "212.4746.92",
+        "IJ2021.2",
+        "IJ183",
+        ij.VersionRange("212.1", "212.*"),
+        arrayOf("java", "org.jetbrains.kotlin:211-1.4.21-release-IJ6693.43")
```

Comment:
@eacheble this need to use the proper kotlin plugin version IJ 212. So this should be `org.jetbrains.kotlin:212-1.4.32-release-IJ2230`.

Reply:
#977 passed I think due to it depending on a more recent version of kotlin. `2.0.x` is stuck with `1.3.x` - probably need to drop the KN support in this branch so we can use most recent version of kotlin.

Reply:
@raniejade I will need your help on this one 🙏

Reply:
I'll take a look this weekend. :)

Reply:
I didn't have time this weekend, unfortunately. Will try to have a look this weekday (night time)